src/pages/api/selectSalesOfferPackage.ts

```diff
@@ -10,12 +10,18 @@ import {
     putUserDataInS3,
 } from './apiUtils/userData';
 import { isSessionValid } from './apiUtils/validator';
-import { SALES_OFFER_PACKAGES_ATTRIBUTE, FARE_TYPE_COOKIE, PERIOD_TYPE_COOKIE } from '../../constants';
+import {
+    SALES_OFFER_PACKAGES_ATTRIBUTE,
+    FARE_TYPE_COOKIE,
+    PERIOD_TYPE_COOKIE,
+    MULTIPLE_PRODUCT_COOKIE,
+} from '../../constants';
 import { NextApiRequestWithSession } from '../../interfaces';
 import { updateSessionAttribute } from '../../utils/sessions';
 
 export interface SelectSalesOfferPackageWithError {
     errorMessage: string;
+    selected: string;
```

Comment:
If we are passing `selected: req.body` in this attribute (line 61), this type would be more than just `selected: string;`?